script/actions/evm.mas.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I feel like this lives in `.github/workflows/config` or something

Reply:
@MarshallOfSound we have some sample configs there but not this one - i can add it to remove this!

Reply:
agreed, done